src/core/ext/filters/client_channel/client_channel.cc

```diff
@@ -1148,6 +1148,13 @@ RefCountedPtr<LoadBalancingPolicy::Config> ChooseLbPolicy(
   } else {
     policy_name = grpc_channel_args_find_string(resolver_result.args,
                                                 GRPC_ARG_LB_POLICY_NAME);
+    if (!LoadBalancingPolicyRegistry::LoadBalancingPolicyExists(policy_name,
```

Comment:
Please remove the TODO below and update the comment to indicate that in the third case, we've verified that above that the policy exists and accepts an empty config.

Reply:
Fixed in the latest commit.